docs/userguide/configuration.rst

```diff
@@ -1923,6 +1923,16 @@ Default: :const:`None`.
 
 See :ref:`routing-options-rabbitmq-priorities`.
 
+.. setting:: task_default_priority
+
+``task_default_priority``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+:brokers: RabbitMQ
```

Comment:
Although this is a indeed a standard feature of RabbitMQ, the Redis transport also emulates priorities.